beacon_node/execution_layer/src/lib.rs

```diff
@@ -1404,18 +1451,183 @@ impl<T: EthSpec> ExecutionLayer<T> {
             "Sending block to builder";
             "root" => ?block_root,
         );
+
         if let Some(builder) = self.builder() {
-            builder
-                .post_builder_blinded_blocks(block)
-                .await
-                .map_err(Error::Builder)
-                .map(|d| d.data)
+            let (payload_result, duration) =
+                timed_future(metrics::POST_BLINDED_PAYLOAD_BUILDER, async {
+                    builder
+                        .post_builder_blinded_blocks(block)
+                        .await
+                        .map_err(Error::Builder)
+                        .map(|d| d.data)
+                })
+                .await;
+
+            match &payload_result {
+                Ok(payload) => info!(
+                    self.log(),
+                    "Builder successfully revealed payload";
+                    "relay_response_ms" => duration.as_millis(),
+                    "block_root" => %block_root,
+                    "fee_recipient" => %payload.fee_recipient,
+                    "block_hash" => %payload.block_hash,
+                    "parent_hash" => %payload.parent_hash
+                ),
+                Err(e) => crit!(
+                    self.log(),
+                    "Builder failed to reveal payload";
+                    "info" => "this relay failure has caused a missed proposal",
```

Comment:
It's possible that the relay still unblinded the block and published it

```suggestion
                    "info" => "this relay failure may cause a missed proposal",
```